src/clr-angular/forms/styles/_variables.forms.scss

```diff
@@ -79,3 +79,11 @@ $clr-forms-radio-top: $clr-forms-checkbox-top;
 $clr-forms-radio-selected-shadow: $clr-forms-checkbox-checked-shadow;
 $clr-forms-radio-focused-shadow: 0 0 $clr-outline-blur $clr-outline-spread $clr-outline-color;
 $clr-forms-radio-disabled-shadow: $clr-forms-checkbox-checked-shadow;
+
+// Range
+$clr-forms-range-progress-fill-color: $clr-color-action-600 !default;
```

Comment:
Custom properties should be created for both of these color variables. See `_properties.forms.scss` for an example.